models/classes/runner/config/QtiRunnerConfig.php

```diff
@@ -38,7 +38,14 @@ class QtiRunnerConfig extends ConfigurableService implements RunnerConfig
 
     const OPTION_CONFIG = 'config';
 
+    /**
```

Comment:
No usages within oat-sa repos, maybe we just remove it in this version?

https://github.com/search?q=org%3Aoat-sa+TOOL_ITEM_THEME_SWITCHER&type=Code

Reply:
Community bundles could use it so it's more safe to mark it as deprecated.